README.md

```diff
@@ -136,24 +136,24 @@ Key features of **Infinity For Reddit**:
 
 ## Contributing
 
-First off, thanks for taking the time to contribute! Contributions are what makes the open source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+First off, thanks for taking the time to contribute! Contributions are what makes the open source community such an amazing place to create, inspire, and learn. Any contributions you make are **greatly appreciated**.
 
-If you have a suggestion that would make this better, please fork the repo and create a pull request.
-It's better to also open an issue describing the issue you want to fix. But it is not required.
+Pease fork the repository and open a pull request if you have a suggestion that would make this better.
```

Comment:
Would being specific about "this" be better? Such as "the application" or "Infinity for Reddit"?

Reply:
Now that you mention it, I think so too. Thanks, it should be good to go now!